metaci/release/models.py

```diff
@@ -17,6 +17,29 @@ def __str__(self):
         return self.name
 
 
+class ImplementationStep(models.Model):
+    release = models.ForeignKey(
+        "release.Release", on_delete=models.CASCADE, related_name="releases"
```

Comment:
related_name should be "implementation_steps"

Then it will be possible to get a queryset for a release object's implementation steps using release.implementation_steps

related_name is the name that's used on the _other_ object.